### DISCUSSION TITLE
MacOS: CMUS not detecting audio files

### DISCUSSION THREAD

**Discussion by rasisin** [cmus/cmus#1158] [Troubleshooting]
I've just installed CMUS in my home directory on my Mac.
I run it from Terminal, go to Browser view, and navigate to the directories containing music files.
* If I navigate inside a folder containing files of supported music formats (a mix of MP3, FLAC, AAC, ALAC, and OGG), none of the files are displayed.
* If I select a folder containing music and press 'a', no music is added to the CMUS library.

I assume this means CMUS is not detecting the music files. What could be the problem and how could it be solved?

Thanks for any suggestions!

**Comment by nefthy** [comment]
What is the output of `cmus --plugins`?

**Comment by rasisin** [comment]
Input Plugins: /usr/local/lib/cmus/ip
  wav:
    Priority: 50
    File Types: wav
    MIME Types:
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue

Output Plugins: /usr/local/lib/cmus/op
  coreaudio
